packages/editor/src/components/media-placeholder/index.js

```diff
@@ -131,7 +131,7 @@ class MediaPlaceholder extends Component {
 			onHTMLDrop = noop,
 			multiple = false,
 			notices,
-			allowedTypes,
+			allowedTypes = [],
```

Comment:
I see that the prop is also used in `onlyAllowsImages` and `onlyAllowsImages` methods, should we add the default values there too?

Reply:
I looked into that earlier, and I don't think changes are necessary in either of those cases. On `onlyAllowsImages` and `onFilesUpload` both, the existing logic should handle both `[]` and `null`/`false`/`undefined`/etc. (`onlyAllowsImages` has the checks in the media placeholder component, while `onFilesUpload` passes the prop value through to be checke within the `mediaUpload` method)